src/soot/jimple/toolkits/callgraph/OnFlyCallGraphBuilder.java

```diff
@@ -82,12 +100,60 @@
  */
 public final class OnFlyCallGraphBuilder
 { 
+	private static final PrimType[] CHAR_NARROWINGS = new PrimType[]{
```

Comment:
These constants are for widening primitive argument types. If there is functionality in the soot codebase that would handle this already I would love to get rid of these.